src/main/java/Calculator.java

```diff
@@ -0,0 +1,37 @@
+package main.java;
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
+        int number1, number2, result;
```

Comment:
Declare variables in one line is bad practice. (Read java convention once again just to summarise)
instead `int a,b,s;`
use
int a;
int b;
int c;

Reply:
changed